model/database/spans.yaml

```diff
@@ -136,6 +136,9 @@ groups:
       classify as errors.
     span_kind: client
     extends: trace.db.common.full
+    attributes:
+      - ref: db.response.returned_rows
+        requirement_level: opt_in  # experimental attributes must be opt-in stable spans
```

Comment:
nit, should we mention in the comment that we want it to be recommended once the attribute is stabilized? We can still dig up the level in the history once we ready, so no strong opinion here

Reply:
ya, good idea, done